docs/how-to/rocgdb.rst

```diff
@@ -0,0 +1,99 @@
+.. meta::
+  :description: This chapter describes the usage of ROCgdb, a tool for debugging
+  ROCm software.
+  :keywords: AMD, ROCm, HIP, ROCgdb, performance, debugging
+
+*******************************************************************************
+ROCgdb
+*******************************************************************************
+
+.. _rocgdb_introduction:
+Introduction
+===============================================================================
+This document introduces ROCgdb, the AMD ROCm debugger for Linux targets.
+
+ROCgdb is an extension to GDB, the GNU Project debugger. The tool provides developers
+with a mechanism for debugging CUDA applications running on actual hardware. This
+enables developers to debug applications without the potential variations introduced
+by simulation and emulation environments. It is meant to present a seamless debugging
+environment that allows simultaneous debugging of both GPU and CPU code within the
+same application, just like programming in HIP is a seamless extension of C++
+programming. The existing GDB debugging features are inherently present for debugging
+the host code, and additional features have been provided to support debugging ROCm
+device code.
+
+ROCgdb supports HIP kernel debugging. It allows the user to set breakpoints, to 
```

Comment:
Perhaps this instead:
It allows you to set breakpoints, single-step ROCm applications, and inspect and modify the memory and variables of any given thread running on the hardware.